src/odemis/driver/picoquant.py

```diff
@@ -1055,8 +2393,8 @@ def PH_CTCStatus(self, i, p_ctcstatus):
         if self._acq_end > time.time():
             ctcstatus.value = 0  # 0 if still running
             # DEBUG
-#             if random.randint(0, 10) == 0:
-#                 raise PHError(-37, PHDLL.err_code[-37])  # bad luck
+            # if random.randint(0, 10) == 0:
+            #    raise PHError(-37, PHDLL.err_code[-37])  # bad luck
```

Comment:
Where these lines supposed to stay in this PR?

Reply:
Yes, that's extra code in case we want to test the error handling when the device communication breaks.